src/Sql/PostgreSQL/Functions/user_create.sql

```diff
@@ -1,34 +1,36 @@
+DROP FUNCTION IF EXISTS user_create(uuid,"varchar","varchar","bool","varchar","varchar","varchar","varchar","text","varchar","text","text","timestamptz","text","text","text","bool","timestamptz","timestamptz","int8","int4","int4","varchar","varchar","varchar","int4","int4","timestamptz","timestamptz");
+
 CREATE OR REPLACE FUNCTION user_create
 (
-    _id                                 uuid,
-    _name                               varchar(50),
-    _email                              varchar(50),
-    _email_verified                      bit,
-    _master_password                    varchar(300),
-    _master_password_hint               varchar(50),
-    _culture                            varchar(10),
-    _security_stamp                     varchar(50),
-    _two_factor_providers               text,
-    _two_factor_recovery_code           varchar(32),
-    _equivalent_domains                 text,
-    _excluded_global_equivalent_domains text,
-    _account_revision_date              timestamptz,
-    _key                                text,
-    _public_key                         text,
-    _private_key                        text,
-    _premium                            bit,
-    _premium_expiration_date            timestamptz,
-    _renewal_reminder_date              timestamptz,
-    _storage                            bigint,
-    _max_storage_gb                     smallint,
-    _gateway                            smallint,
-    _gateway_customer_id                varchar(50),
-    _gateway_subscription_id            varchar(50),
-    _license_key                        varchar(100),
-    _kdf                                smallint,
-    _kdf_iterations                     int,
-    _creation_date                      timestamptz,
-    _revision_date                      timestamptz
+    _id                                 UUID,
+    _name                               VARCHAR,
+    _email                              VARCHAR,
+    _email_verified                     BOOLEAN,
+    _master_password                    VARCHAR,
+    _master_password_hINT               VARCHAR,
+    _culture                            VARCHAR,
+    _security_stamp                     VARCHAR,
+    _two_factor_providers               TEXT,
+    _two_factor_recovery_code           VARCHAR,
+    _equivalent_domains                 TEXT,
+    _excluded_global_equivalent_domains TEXT,
+    _account_revision_date              TIMESTAMPTZ,
+    _key                                TEXT,
+    _public_key                         TEXT,
+    _private_key                        TEXT,
+    _premium                            BOOLEAN,
+    _premium_expiration_date            TIMESTAMPTZ,
+    _renewal_reminder_date              TIMESTAMPTZ,
+    _storage                            BIGINT,
+    _max_storage_gb                     INT,
+    _gateway                            INT,
```

Comment:
also, your types are still not consistent. for example, you use `smallint` in the parameters of the update function below, but you use int here. then you use int for kdf here, but cast it to smallint in the actual query.

I think the way you have it in your update function is correct.